### PR TITLE
Refactor Dashboard Activity 

### PR DESCRIPTION
**Pull request by sagar15795**
I have added following things 

- Refactor Dashboard Activity and  move to ui package  
- Navigation menu selected only for fragments not for dialog
- move all drawer menu title string to string.xml
- change actionbar title on click navigation menu item
- On logout app was finishing but now it is moving to login activity.

### CODE REVIEW

**Comment by sagar15795** [comment]
@therajanmaurya @ianwdunlop  Please review it.

File: app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+import org.apache.taverna.mobile.ui.licence.LicenceFragment;
+import org.apache.taverna.mobile.ui.login.LoginActivity;
+import org.apache.taverna.mobile.ui.myworkflows.MyWorkflowFragment;
+import org.apache.taverna.mobile.ui.workflow.WorkflowFragment;
+import org.apache.taverna.mobile.utils.ActivityUtils;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class DashboardMainActivity extends AppCompatActivity {
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java]: Make DashboardMainActivity -> DashboardActivity.

File: app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_dashboard_main);
+
+        ButterKnife.bind(this);
+
+        if (navigationView != null) {
+            setupDrawerContent(navigationView);
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java]: you do not need to check ```navigationView``` is null or not. 
NavigationView view injected when you called ButterKnife.bind(this);

File: app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+        }
+
+        aboutDialog = new Dialog(this);
+
+        setSupportActionBar(toolbar);
+
+        final ActionBar ab = getSupportActionBar();
+
```
Comment (therajanmaurya) [inline app/src/main/java/org/apache/taverna/mobile/ui/DashboardMainActivity.java]: do not need to give one line gap, when you are setting ActionBar 
```
        setSupportActionBar(toolbar);
        final ActionBar ab = getSupportActionBar();
        if (ab != null) {
            ab.setHomeAsUpIndicator(R.drawable.ic_menu);
            ab.setDisplayHomeAsUpEnabled(true);
        }
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 <resources>
     <string name="os_licence_version_text">Version:</string>
     <string name="os_licence_licence_text">Licence:</string>
-    <string name="os_licences">Licence info</string>
+    <string name="title_nav_os_licences">Licence info</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: Put this String with 
So in future contributor can easily able to find the string and make changes if needed.

**Comment by therajanmaurya** [comment]
@sagar15795 Make the above changes.

**Pull request merged**